src/main/kotlin/no/regnskap/jena/BR.kt

```diff
@@ -0,0 +1,93 @@
+package no.regnskap.jena
+
+import org.apache.jena.rdf.model.ModelFactory
+
+object BR {
+
+    val uri = "http://brreg.no/regnskap#"
+
+    private val model = ModelFactory.createDefaultModel()
+
+    val Regnskap = model.createResource(uri + "Regnskap")
+    val Revisjon = model.createResource(uri + "Revisjon")
+    val Regnskapsprinsipper = model.createResource(uri + "Regnskapsprinsipper")
+    val Virksomhet = model.createResource(uri + "Virksomhet")
+
+    val Eiendeler = model.createResource(uri + "Eiendeler")
+    val Anleggsmidler = model.createResource(uri + "Anleggsmidler")
+    val Omloepsmidler = model.createResource(uri + "Omloepsmidler")
```

Comment:
Ikke ø?

Reply:
Bruker ikke æøå for variabelnavn hvis de spør om application/json eller application application/xml, skal vi endre til æøå hvis de spør om ld+json, rdf+xml eller turtle?